pkg/kube/kube.go

```diff
@@ -31,3 +36,16 @@ var Labels = map[string]string{LabelKey: LabelValue}
 
 // ListOptions is the default set of options to find resources managed by RBAC Manager
 var ListOptions = metav1.ListOptions{LabelSelector: LabelKey + "=" + LabelValue}
+
+// GetClientsetOrDie returns a new Kubernetes Clientset or dies
+func GetClientsetOrDie() *kubernetes.Clientset {
+	kubeConf := config.GetConfigOrDie()
```

Comment:
Consider using `GetConfig` and logging the error if present - IIRC you'll otherwise get an `Exit(1)` without any message logged.

Reply:
Done, good call, thanks!